libafl_nyx/Cargo.toml

```diff
@@ -2,10 +2,18 @@
 name = "libafl_nyx"
 version = "0.1.0"
 edition = "2021"
+authors = ["syheliel <syheliel@gmail.com>"]
+description = "libafl using nyx"
+documentation = "https://docs.rs/libafl_nyx"
+repository = "https://github.com/AFLplusplus/LibAFL/"
+readme = "../README.md"
+license = "MIT OR Apache-2.0"
+keywords = ["fuzzing", "testing", "security"]
+categories = ["development-tools::testing", "emulators", "embedded", "os", "no-std"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-libnyx = {git = "https://github.com/nyx-fuzz/libnyx.git"}
+libnyx = {git = "https://github.com/syheliel/libnyx.git"}
```

Comment:
Why would we want to fork nyx?

Reply:
Some components required in `nyx_bridge.rs` are not `pub`, so I fork a new one and make them accessible

Reply:
fixed